includes/gateways/stripe/includes/admin/admin-actions.php

```diff
@@ -32,6 +32,10 @@ function give_stripe_connect_save_options() {
 
 	$get_vars = give_clean( $_GET );
 
+	if ( ! isset( $get_vars['page'] ) || 'give-settings' !== $get_vars['page'] ) {
```

Comment:
@kjohnson This function runs specifically to save Stripe details when admin returns from the connect server. Can you use `Give_Admin_Setting::is_setting_page` to validate specific tab and section.

Reply:
Cool. I replaced the check.